style/dark-style.css

```diff
@@ -64,9 +92,11 @@ body {
   margin-bottom: 10px;
 }
 
+/*
 .dark nav > ul > li {
   width: 77px;
 }
+*/
```

Comment:
- [ ] Please clean up commented codes in your code base. This makes your code untidy.